src/ca/client/cacChannel.cpp

```diff
@@ -130,6 +130,13 @@ unsigned cacChannel::getHostName (
     return 0u;
 }
 
+unsigned cacChannel::getHostMinorProtocol ( 
+    epicsGuard < epicsMutex > &) const throw ()
+{
+    epicsThreadOnce ( & cacChannelIdOnce, cacChannelSetup, 0);
+    return 99u;
```

Comment:
What is the significance of `99`???

Reply:
I will fix this. 99 was a debug value. I will change this to 0u to indicate an error.